docassemble/MOHUDEvictionProject/data/questions/shared.yml

```diff
@@ -492,23 +499,65 @@ sets:
   - other_parties[i].person_type
 id: names of opposing parties - docket only
 question: |
-  Name of Plaintiff
+  Landlord's name
 subquestion: |
   % if i == 0:
-    You can click on "Parties & Attorneys", the next tab to the right.  This tab should look like this:
+    With your case open on case.net, click the tab labeled "Parties & Attorneys." 
+    It may look like this:
   
     [FILE Parties.png, 100%]
+
+    ${ warn_picture_is_example_template }
     
-    Who is listed as the Plaintiff?
-  % else:
+    Your name should be listed next to the word "Defendant." The landlord's name should be listed
+    next to the word "Plaintiff." The name may look different than it does on your lease.
     
-    [FILE Parties.png, 100%]
-  
-    Another Plaintiff may be listed underneath the first Plaintiff.  Who is the  ${ ordinal(i) } Plaintiff?
+    Who is listed as the first Plaintiff? If there is more than one Plaintiff, you can name them later.
+  % else:
+    Who is the ${ ordinal(i) } Plaintiff listed on your case?
   % endif
 fields:
   - code: |
-      other_parties[i].name_fields(person_or_business='unsure')    
+      other_parties[i].name_fields(person_or_business='unsure')
+---
+template: warn_picture_is_example_template
+content: |
+  <div class="alert alert-warning" role="alert">
+    <i class="fas fa-exclamation-triangle"></i>
+    <strong>Important</strong>: the picture above is just a sample. Make sure you are looking at the results
+    on Missouri Case.net for your case.
+  </div>
+---
+id: signpost casenet for landlord name
+# not (tenant_got_summons and petition_available)
+continue button field: signpost_case_net_for_landlord_name
+question: |
+  Do you want to check your landlord's name online?
+subquestion: |
+  You said you don't have the Summons and Petition with you. If you want, 
```

Comment:
% if tenant_got_summons and petition_available:
  In addition to finding this information in your Summons and Petition, you can check your landlord's name and other information online on a free website run by the Missouri Court.
  % else: